docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
         type: 'string',
         required: true,
         defaultValue: null,
-        description: [
-          'Text to render inside the callout to convey detailed information to the user. The message text has a fixed size.',
-        ],
-        href: '',
+        description: `
+          Main content of the Callout. Content should be localized.
+
+          See [Best Practices](#Best-practices) for more info`,
       },
       {
         name: 'dismissButton',
         type: '{| accessibilityLabel: string, onDismiss: () => void, |}',
-        required: false,
         defaultValue: null,
-        description: [
-          'Callback fired when the dismiss button is clicked (pressed and released) with a mouse or keyboard.',
-          'Supply a short, descriptive label for screen-readers to provide sufficient context about the dismiss button action. IconButtons do not render text for screen readers to read requiring an accessibility label.',
-          'Accessibility: `accessibilityLabel` populates aria-label.',
-        ],
-        href: '',
+        description: `
+          When specified, adds a dismiss button to the Callout. See the [Dismissible variant](#Dismissible) for more info.
+
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).`,
       },
       {
         name: 'iconAccessibilityLabel',
         type: 'string',
         required: true,
         defaultValue: null,
-        description: [
-          'Supply a short, descriptive label for screen-readers to provide sufficient context about the callout intent.',
-          'The callout component displays `error`, `info`, or `warning` icons which visually indicate the intent of the component. `iconAccessibilityLabel` allows screen readers to communicate that intent.',
-          'Accessibility: It populates aria-label.',
-        ],
-        href: '',
+        description: `
+          Label to describe the icon’s purpose
```

Comment:
This format is funky. Should be two sentences inline, with periods at their ends?